gulpfile.js/tasks/release.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const gulp = require('gulp')
+const runSequence = require('run-sequence')
+
+gulp.task('release', ['build'], (done) => {
+  runSequence(
+    'changelog',
+    'v3',
+    'zip',
+    'v4',
+    'zip',
```

Comment:
Could `zip` just run on any/all dirs in `dest`? That way we could run `v3, v4, zip`

Reply:
Producing one zip you mean?